projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.ts

```diff
@@ -21,38 +21,20 @@ export class MerchandisingCarouselComponent {
     protected componentData: CmsComponentData<
       CmsMerchandisingCarouselComponent
     >,
-    protected merchandisingCarouselComponentService: MerchandisingCarouselComponentService
+    protected merchandisingCarouselComponentService: MerchandisingCarouselComponentService,
+    protected el: ElementRef
```

Comment:
I'd prefer hostbinding (no constructor pollution), but css variables are sanitized in angular 8 (which isi fixed in ng9). So this solution seems fine!